build.gradle

```diff
@@ -322,3 +322,20 @@ tasks.register('installGitHooks', Copy) {
 tasks.register('downloads') {
 	dependsOn allprojects*.tasks*.withType(VerifiedDownload)
 }
+
+task aggregatedJavadocs(type: Javadoc, description: 'Generate javadocs from all child projects as if it was a single project', group: 'Documentation') {
```

Comment:
@liblit do i need to do some kind of lazy configuration thing here?

Reply:
Not absolutely required, but yes, using the [task configuration avoidance APIs](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html) would be preferred. Replace the `task` line with the following:

``` gradle
tasks.register('aggregatedJavadocs', Javadoc) {
	description = 'Generate javadocs from all child projects as if it was a single project'
	group = 'Documentation'
```